### ISSUE TITLE
Take into account Restlet Framework documentation issues

### ISSUE THREAD

**Issue by jlouvel** [restlet/restlet-tutorial#3]
See the list of doc issues:
https://github.com/restlet/restlet-framework-java/issues?labels=Documentation&page=1&state=open

**Comment by marti1125** [comment]
Hi I found an issue about tutorial 

http://restlet.com/technical-resources/restlet-framework/tutorials/2.3#part01

@Get("json") 
public String toString() {
        return "hello, world";
}

**Comment by thboileau** [comment]
Hi @marti1125 , if I understand well, you point the fact that the `toString` method is simply annotated with the @Get annotation whithout any parameter, whereas you would like to add the "json" parameter?

**Comment by marti1125** [comment]
yes it works but I see an error so I decided to add "json" http://xhtml.com/8C236CAC-543D-48B1-A9B1-F1D996296A2A/xml-parsing-error.gif

**Comment by thboileau** [comment]
Thanks @marti1125 , I've added the "txt" parameter which is better than "json".

**Comment by marti1125** [comment]
:+1: how to I can deploy a maven java app ?
